commit d8792c7d3d35b1b306670b8e84512047448bd3da
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Tue Oct 23 15:40:47 2018 +0100

    ASoC: googlevoicehat-codec: Make driver function structure const
    
    Make voicehat_component_driver a const structure.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/sound/soc/bcm/googlevoicehat-codec.c b/sound/soc/bcm/googlevoicehat-codec.c
index cc22c7723474..61e97c382907 100644
--- a/sound/soc/bcm/googlevoicehat-codec.c
+++ b/sound/soc/bcm/googlevoicehat-codec.c
@@ -82,7 +82,7 @@ static const struct snd_soc_dapm_route voicehat_dapm_routes[] = {
 	{"Speaker", NULL, "HiFi Playback"},
 };
 
-static struct snd_soc_component_driver voicehat_component_driver = {
+static const struct snd_soc_component_driver voicehat_component_driver = {
 	.probe = voicehat_component_probe,
 	.remove = voicehat_component_remove,
 	.dapm_widgets = voicehat_dapm_widgets,
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1216_d8792c7d3d35b1b306670b8e84512047448bd3da.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1216_d8792c7d3d35b1b306670b8e84512047448bd3da.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
